src/webui/api/synccontroller.cpp

```diff
@@ -418,6 +418,11 @@ void SyncController::maindataAction()
     sessionManager()->session()->setData(QLatin1String("syncMainDataLastAcceptedResponse"), lastAcceptedResponse);
 }
 
+void SyncController::statsAction()
```

Comment:
You misuse Sync Controller. It's for sending delta-compressed data.

Reply:
I can move it to `AppController`, but then I'd have to externalize `getTranserInfo()` in `SyncController`.

Reply:
>I can move it to AppController

AppController is for other things.
There is TransferController existed. You can use it. Or you can implement valid "sync" action here. Since you want to get the data frequently it's better way for you.

Reply:
I don't think this info make sense to live in `TransferController`, since transfer info is a small subset of what it provides.

> Or you can implement valid "sync" action here. Since you want to get the data frequently it's better way for you.

Maybe, but that's definitely overkill for an api that gets hit every 5 seconds, produces a very small amount of data, and only gets called when the stats window is open (aka almost never).